README.md

```diff
@@ -6,36 +6,111 @@
 [![codecov](https://codecov.io/gh/ZcashFoundation/zebra/branch/main/graph/badge.svg)](https://codecov.io/gh/ZcashFoundation/zebra)
 ![License](https://img.shields.io/badge/license-MIT%2FApache--2.0-blue.svg)
 
-### 🚧 UNDER CONSTRUCTION 🚧
+## About
 
 [Zebra](https://zebra.zfnd.org/) is the Zcash Foundation's independent,
 consensus-compatible implementation of the Zcash protocol, currently under
 development.  Please [join us on Discord](https://discord.gg/na6QZNd) if you'd
 like to find out more or get involved!
 
+## Alpha Releases
+
+Our first alpha release is planned for December 8th, 2020.
+
+The goals of this release are to:
+- participate in the Zcash network,
+- replicate the Zcash chain state, and
+- implement the Zcash proof of work consensus rules.
+
+### Getting Started
+
+Run `cargo ...` **TODO**
```

Comment:
Rust experts: what's the right command to download and build Zebra from a git tag?

Reply:
Also list dependencies for rocksdb and zcash_script

Reply:
probably `cargo install --git https://github.com/ZcashFoundation/zebra.git --tag <release-tag>`